examples/with-amp/pages/_document.js

```diff
@@ -0,0 +1,29 @@
+import Document, { Head } from 'next/document'
+import { DOMProperty } from 'react-dom/../lib/ReactInjection'
+
+DOMProperty.injectDOMPropertyConfig({
```

Comment:
It doesn't trigger errors but it's not a valid AMP document anymore. Try going to http://localhost:3000/#development=1 and check the console